bsp/lpc55sxx/lpc55s69_nxp_evk/rtconfig.py

```diff
@@ -0,0 +1,161 @@
+import os
+import sys
+
+# toolchains options
+ARCH='arm'
+CPU='cortex-m4'
```

Comment:
please set the value cpu as `libcpu\arm\cortex-m33`

Reply:
Or is it currently only temporarily supporting cortex-m4?

Reply:
as the libcpu\arm\cortex-m33 is supporting by tangyuxin, we have use cortex-m4 instead of m33, once the new libcpu integrate into RT-Thread, we may set this value as libcpu\arm\cortex-m33